doc/man3/RAND_add.pod

```diff
@@ -54,6 +57,13 @@ should consider using L<RAND_load_file(3)> instead.
 
 RAND_seed() is equivalent to RAND_add() with B<randomness> set to B<num>.
 
+RAND_keep_random_devices_open() is used to control file descriptor use by
+the random seed sources. Some seed sources maintain open file descriptors
+by default, which allows such sources to operate in a chroot(2) jail
+without the associated device nodes being available. When the B<keep> argument
+is zero, this call disables the retention of file descriptors.  Conversely,
+a non-zero argument enables the retention of file descriptors.
```

Comment:
As I understand it this only has an effect the next time we get more entropy from the seed sources, i.e. calling RAND_keep_random_devices_open(0), would not immediately close already open file descriptors. Should we say something about that?

Reply:
> RAND_keep_random_devices_open(0), would not immediately close already open file descriptors.

False, the file descriptors are closed immediately in [rand_pool_keep_random_devices_open()](https://github.com/openssl/openssl/pull/6432/files#diff-9eb3297e1ad6f861ae690c543e5118e9R385).

Reply:
Ah - right! My misunderstanding. In that case, do we need to say that this happens immediately?

Reply:
The normal use case would be to call ` RAND_keep_random_devices_open()` only once during initialization before the first randomness is generated (in which case it makes no difference how it's implemented). You normally don't flip this feature on and off. So maybe better add a sentence that this should be called during early initialization (unless one is happy with the default).

Reply:
I've added a sentence and rebased to avoid merge conflicts.